lib/create-plugin.js

```diff
@@ -412,6 +412,7 @@ function getDeepestPathAndQueryArguments(definition) {
       // When no selections returned from filtering, deepest path is found
       if (filtered.length === 0 || filtered.length > 1) {
         foundDeepestPath = true
+        deepestPath = parts
```

Comment:
It has been funny to see how all these fixes have been very slight changes. But maybe that's a good sign.

Reply:
yea i may also open a follow up PR to extract some of this logic. there's a lot going on and hard to grok unless you've been in here before